src/encoded/static/components/static-pages/components/StackedBlockVisual.js

```diff
@@ -526,11 +583,43 @@ export class StackedBlockGroupedRow extends React.PureComponent {
         });
     }
 
+    //The function of creating sorting according to the total values of the data
+    //As a result, objects in the array are created. (groupingKey, count)
+    //For example ('groupingKey' :dataBinding, 'count': 146)
+    static sortBlock = memoize(function (data, groupedDataIndices, groupingKey, sortField) {
```

Comment:
If we have multiple matrices (or StackedBlockGroupedRow) on a single page/view, then would make sense to have this static sortBlock function not be memoized here (as static funcs are global and would be shared between all the matrix instances, negating the effect if more than 1 instance of the matrix is present), but instead to have per-instance memoization like `this.memoized = { sortBlock: memoize(StackedBlockGroupedRow. sortBlock) }`.

Reply:
Thanks for the feedback. I moved memoization into the constructor. (https://github.com/4dn-dcic/fourfront/pull/1488/commits/227347d2d4c6f69659386c4e50c8377f2219a2a4)